ppr-ui/src/composables/useParty.ts

```diff
@@ -21,7 +21,11 @@ export const useParty = () => {
 
   const getFormattedBirthdate = (party: PartyIF): string => {
     if (party.birthDate) {
-      const date = new Date(party.birthDate)
+      let date = new Date(party.birthDate)
+      // prod hotfix to check if browser does not support locales
+      if (!(date instanceof Date && !isNaN(date.valueOf()))) {
+        date = new Date(party.birthDate.substring(0, party.birthDate.indexOf('T')))
+      }
```

Comment:
Condition looks fine to me!
With regards to the conversion, future state could look into `convertDate` in `date-helper.ts`. 
Not sure if applicable but worth a look at a later date. 

Does this date value format suffice for any api validations on this property? ie this conversion files without issue?

Reply:
I haven't tried the full flow. This fix only for the UI formatting/display. We do not update the date in the `party.birthDate`.

Reply:
Sounds good. 
I recall Doug saying the data looked acceptable in the db, so if we don't mutate the original values, should be low risk!